### PR TITLE
Updates for Terraform 0.13 - Remove module-wide create/enable variable

### PR DESCRIPTION
**Pull request by cmassey-berico**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -21,7 +17,6 @@ resource "aws_vpn_gateway" "this" {
 }
 
 resource "aws_customer_gateway" "this" {
-  count = var.create_vpn_connection ? length(var.cgw_ip_addresses) : 0
```
Comment (lorengordon) [inline main.tf:24]: We're only removing `var.create_vpn_connection`. We still need a customer gateway per CGW IP.

File: main.tf [hunk 2]
```diff
@@ -35,7 +30,6 @@ resource "aws_customer_gateway" "this" {
 }
 
 resource "aws_vpn_connection" "this" {
-  count = var.create_vpn_connection ? length(var.cgw_ip_addresses) : 0
```
Comment (lorengordon) [inline main.tf:38]: Same idea, still need a VPN connection per CGW IP...

File: main.tf [hunk 3]
```diff
@@ -35,7 +30,6 @@ resource "aws_customer_gateway" "this" {
...
 
   vpn_gateway_id      = aws_vpn_gateway.this[0].id
```
Comment (lorengordon) [inline main.tf:34]: We are removing `count` entirely from `aws_vpn_gateway`, so the resource is no longer a set and we need to stop indexing into it...
```suggestion
  vpn_gateway_id      = aws_vpn_gateway.this.id
```

File: main.tf [hunk 4]
```diff
@@ -50,14 +44,12 @@ resource "aws_vpn_connection" "this" {
 }
 
 resource "aws_vpn_connection_route" "this" {
-  count = var.create_vpn_connection ? length(var.destination_cidr_blocks) : 0
```
Comment (lorengordon) [inline main.tf:53]: Still need to count over destination cidr blocks

File: main.tf [hunk 5]
```diff
@@ -50,14 +44,12 @@ resource "aws_vpn_connection" "this" {
...
 
   destination_cidr_block = var.destination_cidr_blocks[count.index]
   vpn_connection_id      = aws_vpn_connection.this[0].id
 }
 
 resource "aws_vpn_gateway_route_propagation" "this" {
-  count = var.create_vpn_connection ? var.propagating_route_table_count : 0
```
Comment (lorengordon) [inline main.tf:60]: Still need to count over var.propagating_route_table_count

File: outputs.tf [hunk 6]
```diff
@@ -1,6 +1,6 @@
 output "vpn_gateway_id" {
   description = "ID of the VPN Gateway"
-  value       = join("", aws_vpn_gateway.this.*.id)
+  value       = aws_vpn_gateway.this.*.id
```
Comment (lorengordon) [inline outputs.tf:3]: the `.*` is part of the join syntax, and an artifact of previously using count on aws_vpn_gateway.
```suggestion
  value       = aws_vpn_gateway.this.id
```

**Review by lorengordon — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (cmassey-berico) [reply main.tf:24]: Gotcha. So, the conditional is being removed entirely? 

`count = length(var.cgw_ip_addresses)`

File: main.tf (hunk 2 above)
Comment (cmassey-berico) [reply main.tf:38]: `count = length(var.cgw_ip_addresses)` ?

File: main.tf (hunk 4 above)
Comment (cmassey-berico) [reply main.tf:53]: `count = length(var.destination_cidr_blocks)` ?

File: main.tf (hunk 5 above)
Comment (cmassey-berico) [reply main.tf:60]: `count = var.propagating_route_table_count` ?

File: main.tf (hunk 1 above)
Comment (lorengordon) [reply main.tf:24]: That's correct, yes. I should have left it as a suggestion, sorry...

File: main.tf (hunk 2 above)
Comment (lorengordon) [reply main.tf:38]: Exactly

**Comment by lorengordon** [comment]
@cmassey-berico Please also run `terraform fmt -recursive`

**Review by lorengordon — APPROVED** [review]
Went ahead and made the format/readme changes myself. Results of `make test`:

```
PASS
ok      terraform-aws-tardigrade-vpn-connection/tests   486.644s
[terratest/test]: Completed successfully!
```

**Pull request merged**